boards/adafruit/itsybitsy/adafruit_itsybitsy_nrf52840.dts

```diff
@@ -8,6 +8,7 @@
 #include <nordic/nrf52840_qiaa.dtsi>
```

Comment:
I think this commit would be better earlier in the series, right after "dts: bindings: led_strip: Add common binding". This would reduce the risk of breaking a build in case of a bisection.

Reply:
Moved